feat: errors

Abdur-rahmaanJ opened the pull request.

Abdur-rahmaanJ committed:
feat: errors

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/shopyo/shopyo?pullRequest=145) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/shopyo/shopyo?pullRequest=145) before we can accept your contribution.<br/>**1** out of **2** committers have signed the CLA.<br/><br/>:white_check_mark: Abdur-rahmaanJ<br/>:x: pre-commit-ci[bot]<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/shopyo/shopyo?pullRequest=145) it.</sub>

Abdur-rahmaanJ committed:
Fix/sqla version (#146)

* [pre-commit.ci] pre-commit autoupdate (#141)

updates:
- [github.com/asottile/pyupgrade: v3.8.0 → v3.10.1](https://github.com/asottile/pyupgrade/compare/v3.8.0...v3.10.1)
- [github.com/psf/black: 23.3.0 → 23.7.0](https://github.com/psf/black/compare/23.3.0...23.7.0)

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

* fix: bound sqla versions

---------

Co-authored-by: pre-commit-ci[bot] <66853113+pre-commit-ci[bot]@users.noreply.github.com>

Abdur-rahmaanJ committed:
feat: Add error on missing installed_package
